.github/workflows/ci.yml

```diff
@@ -8,34 +8,45 @@ on:
 
 jobs:
   build:
-    timeout-minutes: 60
     strategy:
       fail-fast: false
       matrix:
         target:
           - os: linux
             cpu: amd64
+          - os: linux-gcc-14 # This is to use ubuntu 24 and install gcc 14. Should be removed when ubuntu-latest is 26.04
+            cpu: amd64
           - os: linux
             cpu: i386
           - os: macos
             cpu: amd64
+          - os: macos
+            cpu: arm64
           - os: windows
             cpu: amd64
-          #- os: windows
-            #cpu: i386
-        branch: [version-1-6, version-2-0, devel]
+        branch: [version-2-0, version-2-2, devel]
```

Comment:
non-directly-eth libraries should continue testing 1.6 so that we know when / if it breaks